package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "Activate VSCode Icon Theme",
```

Comment:
What do you mean by that? They rely on the prefix. I would stick to `Activate Icon Theme` or `Activate VSCode Icons` or `Activate VSCode Icons Icon Theme`.